spec/system/case_contacts/index_spec.rb

```diff
@@ -1,28 +1,41 @@
 require "rails_helper"
 
 RSpec.describe "case_contacts/index", :disable_bullet, type: :system do
-  let(:organization) { create(:casa_org) }
   let(:volunteer) { create(:volunteer, display_name: "Bob Loblaw", casa_org: organization) }
-  let(:casa_case) { create(:casa_case, casa_org: organization, case_number: "CINA-1") }
-  let!(:case_assignment) { create(:case_assignment, volunteer: volunteer, casa_case: casa_case) }
-  let!(:case_contact) { create(:case_contact, creator: volunteer, casa_case: casa_case) }
+  let(:organization) { create(:casa_org) }
 
-  before(:each) do
-    sign_in volunteer
-    visit case_contacts_path
-  end
+  context "with case contacts" do
+    let(:casa_case) { create(:casa_case, casa_org: organization, case_number: "CINA-1") }
```

Comment:
I'm pretty sure `case_number: "CINA-1"` was to satisfy that regex rule that was deleted and is not needed anymore